src/lib.rs

```diff
@@ -9,3 +12,10 @@ pub fn make_static<'a, T>(input: &'a T) -> &'static T {
     let f: fn(_, &'a T) -> &'static T = helper;
     f(&&(), input)
 }
+
+/// Use black magic fuckery to change the lifetime on a mutable refrence
+/// May introduce undefined behavior.
+pub fn make_static_mut<'a, 'b, T>(input: &'a mut T) -> &'b static T {
+    let f: fn(_, &'a mut T) -> &'b static T = helper_mut;
```

Comment:
This isn’t valid syntax; you mean `&'b mut T`.

```suggestion
pub fn make_static_mut<'a, 'b, T>(input: &'a mut T) -> &'b mut T {
    let f: fn(_, &'a mut T) -> &'b mut T = helper_mut;
```

Reply:
Fixed.